staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -589,6 +589,21 @@ type KubeletConfiguration struct {
 	// Default: 5
 	// +optional
 	ContainerLogMaxFiles *int32 `json:"containerLogMaxFiles,omitempty"`
+
+	// ContainerLogMaxWorkers specifies the maximum number of concurrent workers to spawn
+	// for performing the log rotate operations. Set this count to 0 for disabling the
+	// concurrent log rotation workflows
+	// Default: 0
```

Comment:
this should match the number of workers and default to 1 to be consistent with other worker count configs / flags

Reply:
add validation to ValidateKubeletConfiguration that a value >= 1 is specified here

Reply:
Ack.